l1-contracts/test/portals/TokenPortal.sol

```diff
@@ -28,21 +28,27 @@ contract TokenPortal {
    * @param _amount - The amount to deposit
    * @param _deadline - The timestamp after which the entry can be cancelled
    * @param _secretHash - The hash of the secret consumable message
+   * @param _canceller - The address that can cancel the L1 to L2 message.
+   * The message content is hashed with this address to prevent anyone else from cancelling the message.
```

Comment:
Don't think it need the extra informaiton in the comments here. Namely "the message content" is not clear for people just looking at this code, so the not clear what the others parts of it is. Think it is fine with just the `@param` part of it.